### DISCUSSION TITLE
avalonia datagrid advices

### DISCUSSION THREAD

**Discussion by ghost** [AvaloniaUI/Avalonia#10984] [Q&A]
Hello, 
I'm new on AvaloniaUI; 
I need to do a little application that show and update, form a posgresql db every 10 sec, a datagrid like this
![immagine](https://user-images.githubusercontent.com/82973069/231118291-6538708c-4870-44e6-aaf0-6c56b12ef61d.png)
Since some cell are merged it could be done?
I read that it would be possible to run an AvaloniaUI application on RPI without loading X server; Since RPI is very hard to find in my country this days, could I done the same with RockPi devices?
Any advice?

**Comment by maxkatz6** [comment]
> Since some cell are merged it could be done?

You can do the same with DataGridTemplateColumn

> I read that it would be possible to run an AvaloniaUI application on RPI without loading X server; Since RPI is very hard to find in my country this days, could I done the same with RockPi devices?

If you can run arm linux on RockPi with dotnet installed, then it should be possible.
